tests/unit/ChangePasswordControllerTest.php

```diff
@@ -21,17 +23,41 @@
 use Test\TestCase;
 
 class ChangePasswordControllerTest extends TestCase {
+	/** @var  IRequest | \PHPUnit_Framework_MockObject_MockObject */
+	private $request;
+	/** @var  IL10N | \PHPUnit_Framework_MockObject_MockObject */
+	private $l10n;
+	/** @var  IUserSession | \PHPUnit_Framework_MockObject_MockObject */
+	private $userSession;
+	/** @var  IUserManager | \PHPUnit_Framework_MockObject_MockObject */
+	private $userManager;
+	/** @var  IGroupManager | \PHPUnit_Framework_MockObject_MockObject */
+	private $groupManager;
+	/** @var  IMailer | \PHPUnit_Framework_MockObject_MockObject */
+	private $mailer;
+	/** @var  IConfig | \PHPUnit_Framework_MockObject_MockObject */
+	private $config;
+	/** @var  ChangePasswordController */
+	private $controller;
+	public function setUp() {
```

Comment:
Not really important in this particular case, but we should keep the same visibility of the overwritten method unless there is a good reason to change the visibility.